app/views/content/cookies.html.erb

```diff
@@ -1,140 +1,138 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Cookies</h1>
-
-    <h2 class="govuk-heading-l" id="cookies-that-measure-website-use">Cookies that measure website use</h2>
-
-    <p class="govuk-body">
-    We use Google Analytics software to collect information about how you use ‘<%= service_name %>’. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.
+    <h1 class="govuk-heading-l">Cookies</h1>
```

Comment:
We’re currently using `xl` for headings on Apply, so we can revert this change.

```suggestion
    <h1 class="govuk-heading-xl">Cookies</h1>
```

Reply:
👍